### PR TITLE
add Simulate Host Faults and update chaosd overview

### PR DESCRIPTION
**Pull request by TomShawn**
Signed-off-by: TomShawn <41534398+TomShawn@users.noreply.github.com>

This PR:

+ translates `simulate-host-console-in-physical-nodes.md` from https://github.com/chaos-mesh/website-zh/pull/26 and https://github.com/chaos-mesh/website-zh/pull/55
+ updates `chaosd-overview.md` from https://github.com/chaos-mesh/website-zh/pull/40.

### CODE REVIEW

File: docs/simulate-host-console-in-physical-nodes.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Create a host shutdown experiment
+
+To create a host shutdown experiment, run the following command:
+
+```bash
+chaosd attack host shutdown
+```
+
+The output is as follows:
```
Comment (qiancai) [inline docs/simulate-host-console-in-physical-nodes.md:39]:
```suggestion
The example output is as follows:
```

**Review by qiancai — APPROVED** [review]
Good translation. I have only one minor comment. PTAL. @TomShawn

File: docs/chaosd-overview.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ sidebar_label: Chaosd
 
 ## Chaosd Introduction
```
Comment (Liuxiaozhen12) [inline docs/chaosd-overview.md:6]:
```suggestion
## Chaosd introduction
```

**Review by Liuxiaozhen12 — APPROVED** [review]
rest LGTM

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Pull request merged**